udata/templates/metadata.html

```diff
@@ -41,4 +41,5 @@
 />
 {% endif %}
 
-<meta name="notify-in" content="{{ meta.notify_in or '#notifications-container' }}">
+<meta name="notify-in" content="{{ meta.notify_in or '#notifications-container' }}" />
+<meta name="check-urls" content="{{ 'true' if config.CROQUEMORT else 'false' }}" />
```

Comment:
Maybe `str(bool(config.CROQUEMORT)).lower()` mmh typing it I'm not sure anymore it's more explicit…